pages/4 Depth Cost.py

```diff
@@ -0,0 +1,57 @@
+import streamlit as st
+import plotly.express as px
+import pandas as pd
+import numpy as np
+from utils.queries import Subgraph
+from utils.market_makers import StableSwapBinary
+
+st.title("Depth Cost")
+
+pool_id = st.text_input('Pool id', value='0x2d011adf89f0576c9b722c28269fcb5d50c2d17900020000000000000000024d')
+
+if st.session_state.get("pool_id") != pool_id:
```

Comment:
using session_state with same name variables seems to be a problem, but using the documentation initializing process creates a problem when the changing tabs. Suggestion: just change to  different names without initializing

Reply:
Bug description:

In page 'Depth cost' change pool id to: 0x178e029173417b1f9c8bc16dcec6f697bc32374600000000000000000000025d, then change to another tab inside the app (eg. Stable curve). Come back to 'Depth cost' and it shows the default pool id but with 0x178e029173417b1f9c8bc16dcec6f697bc32374600000000000000000000025d data